apis/io-engine/protobuf/v1/host.proto

```diff
@@ -56,6 +56,8 @@ message BlockDevice {
   Partition partition = 9;      // partition information in case where device represents a partition
   Filesystem filesystem = 10;   // filesystem information in case where a filesystem is present
   bool available = 11;          // identifies if device is available for use (ie. is not "currently" in use)
+  string connectiontype = 12;   // the type of bus through which the device is connected to the system
+  string isrotational = 13;     // indicates whether the device is rotational or non-rotational (1 or 0)
```

Comment:
```suggestion
  bool is_rotational = 13;     // indicates whether the device is rotational or non-rotational
```

Reply:
The queue/rotational attribute may or may not be present. Should I make it optional<bool>?

Reply:
Yes that makes sense

Reply:
Made it optional bool